deploy/olm-catalog/service-telemetry-operator/1.1.0/service-telemetry-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -15,6 +15,8 @@ metadata:
             "graphingEnabled": false,
             "highAvailabilityEnabled": false,
             "metricsEnabled": true,
+            "snmpTrapsEnabled": false,
+            "snmpTrapsTarget": "192.168.24.254",
```

Comment:
Sorry I think I was wrong before or misunderstanding something pre-coffee. This looks right and is what I'd expect.

The only other thing is to add the `specDescriptors` to provide the UI hints allowing the Operator to be configured from the OpenShift console.